src/main/java/org/sonarsource/scanner/maven/SonarQubeMojo.java

```diff
@@ -95,17 +95,14 @@
   @Component
   private RuntimeInformation runtimeInformation;
 
-  @Component
+  @Parameter(defaultValue = "${mojoExecution}", readonly = true, required = true)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder annotation arguments to match the order of declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3340)